cookbook/controller/error_pages.rst

```diff
@@ -71,8 +62,29 @@ templates is based on the HTTP status code and the request format:
 .. _overriding-or-adding-templates:
 
 To override these templates, simply rely on the standard Symfony method for
-:ref:`overriding templates that live inside a bundle <overriding-bundle-templates>`.
-For example, to override the 404 error template for HTML pages, create a new
+:ref:`overriding templates that live inside a bundle <overriding-bundle-templates>`:
+put them in the ``app/Resources/TwigBundle/views/Exception/`` directory.
+
+A typical project that returns HTML and JSON pages, might look like this:
+
+.. code-block:: text
+
+    app/
+    ├─ Resources/
+    │  └─ TwigBundle/
+    │     └─ views/
+    │        └─ Exception/
+    │           ├─ error404.html.twig
+    │           ├─ error403.html.twig
+    │           ├─ error.html.twig      # All other HTML errors (including 500)
+    │           ├─ error404.json.twig
+    │           ├─ error403.json.twig
+    │           ├─ error.json.twig      # All other JSON errors (including 500)
```

Comment:
I don't it's usefull to show the line from `app/`, we don't do it for the sub dirs